src/ext/coms.c

```diff
@@ -6,13 +6,15 @@
 #include <string.h>
 
 #include "coms.h"
+#include "coms_curl.h"
 #include "env_config.h"
 #include "mpack.h"
 #include "vendor_stdatomic.h"
 
 typedef uint32_t group_id_t;
 
 #define GROUP_ID_PROCESSED (1UL << 31UL)
+#define MEMORY_PRESSURE_LEVEL (0.8)
```

Comment:
Is there a reason why we're going for 80% vs some other percentage? Maybe we could add a comment to clarify? :)

Reply:
I've made this value configurable, and added comment clarifying the arbitrary nature of it :D